src/components/Admin/ListUser/ListUser.js

```diff
@@ -184,6 +188,56 @@ export default class ListUser extends Component {
     });
   };
 
+  handleSearch = value => {
+    this.setState({
+      search: true,
+    });
+    let url;
+    url =
+      `${urls.API_URL}/aaa/getUsers.json?access_token=` +
+      cookies.get('loggedIn') +
+      '&search=' +
+      value;
+    $.ajax({
+      url: url,
+      dataType: 'jsonp',
+      jsonpCallback: 'pvsdu',
+      jsonp: 'callback',
+      crossDomain: true,
+      success: function(response) {
+        let userList = response.users;
+        let users = [];
+        userList.map((data, i) => {
+          let user = {
+            serialNum: ++i,
+            email: data.name,
+            confirmed: data.confirmed,
```

Comment:
No point of storing this if you're changing it later anyways.

Reply:
Removed it